detektive/src/test/kotlin/cash/detektive/javacompat/AnnotatePublicApisWithJvmOverloadsTest.kt

```diff
@@ -46,9 +85,26 @@ internal class AnnotatePublicApisWithJvmOverloadsTest(private val env: KotlinCor
 
   companion object {
     data class ErrorTestCase(
-      val description: String, val elementType: ElementType, val elementName: String, val code: String
+      val description: String,
+      val elementType: ElementType,
+      val elementName: String,
+      val code: String
     )
 
+    private var defaultLoggerFactory = Logger.getFactory()
+
+    @BeforeAll
+    @JvmStatic
+    fun beforeAll() {
+      Logger.setFactory { PrintingLogger(System.out) }
```

Comment:
The IJ default logger throws an exception on errors which makes the build fails. I'm replacing that logger for this test here.